common/index.html

```diff
@@ -51,6 +51,10 @@ <h2>Your Browser is Out of Date</h2>
           MyCrypto requires certain features that your browser doesn't offer. Your browser may also
           be missing security updates that could open you up to vulnerabilities. Please update your
           browser, or switch to one of the following browsers to continue using MyCrypto.
+          
```

Comment:
HTML ignores extra whitespace characters, so it would appear that "If you're using..." is part of the same paragraph even though there is a blank line in the code. Instead, you need to start a new `<p>` (paragraph) tag like this:

```suggestion
</p>
<p class="is-desktop">
```